### DISCUSSION TITLE
Investigate --api option

### DISCUSSION THREAD

**Discussion by devmsh** [tighten/lambo#165] [Ideas]
Given the release of Laravel Jetstream (Inertia/Livewire) and tools like [Laravel Blueprint](https://blueprint.laravelshift.com/), what does Laravel API development look like in 2020 and how can Lambo best support it 🤔 ?

**Original issue description from @devmsh**:

> It sill be awesome if you add --api option than default the new Laravel project to the best options for a fully API application that did not contain any sort of UI.

**Comment by jonsugar** [comment]
@devmsh What kind of setup do you have in mind for a `--api` option? Is it just an alias for running a combination of the existing options or is there other configuration not yet supported you have in mind?

**Comment by devmsh** [comment]
I think --all - any ui related option will be fine

**Reply by jonsugar** [reply]
@devmsh do you mean Lambo would remove all the UI from the application?

**Comment by jonsugar** [comment]
@devmsh sorry that we have not paid any attention to this for a long time.  Is this something you are still interested in or think would be useful?

If yes, can you be give more detail about what Lambo would do if `--api` was specified. Specifically:

- What does 'best options for a fully API application' mean?
